orcid-persistence/src/main/resources/db-master.xml

```diff
@@ -228,6 +228,8 @@
 	<include file="/db/updates/identifier-types/remove-cv-id.xml" />
 	<include file="/db/updates/add_column_reason_locked.xml" />
 	<include file="/db/updates/add_column_reason_locked_description.xml" />
+	<include file="/db/updates/add_column_record_locked_date.xml" />
+	<include file="/db/updates/add_column_record_locked_admin_id.xml" />
```

Comment:
@auumgn just to be a really pain ... could you please keep just 1 file for this? You just need to merge the two files by keeping both `<changeSet>` , and then please format the code.

Thanks